### PR TITLE
added PR conventions

### PR DESCRIPTION
**Pull request by yomaokobiah**

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -70,3 +72,26 @@
...
 * [Git cherry-pick](https://git-scm.com/docs/git-cherry-pick)
+<br>
+
+## PR review rules
+ * Input from two reviewers is needed to merge a pull request. One reviewer approves the merge and the other reviewer merges the pull request.
+<br>
+
+## Issues
+### New issue
+Discovering an issue is great, here's what you need to do when you discover an issue;
```
Comment (galuhsahid) [inline contributing.md]: I think "an issue;" should be "an issue:"

File: contributing.md [hunk 2]
```diff
@@ -70,3 +72,26 @@
 ### Reference
 * [GitHub forking](https://gist.github.com/Chaser324/ce0505fbed06b947d962)  
 * [Git cherry-pick](https://git-scm.com/docs/git-cherry-pick)
+<br>
```
Comment (galuhsahid) [inline contributing.md]: We probably don't need a `<br>` here, I think leaving the line blank would give the same effect

File: contributing.md [hunk 3]
```diff
@@ -70,3 +72,26 @@
...
+Discovering an issue is great, here's what you need to do when you discover an issue;
+* Search if the issue has already been created.
+* If yes and open refer to existing issue.
+* If yes and closed reopen issue with descriptive comment.
+* If no, create the issue by;
+   * Following our issue [template.](https://github.com/pandas-dev/pandas/blob/master/.github/ISSUE_TEMPLATE.md)
+### Existing issue
+* Read comments.
+* Find out if anyone is working on it, if no, offer to do it. If yes, see if you can be of help.
+<br>
```
Comment (galuhsahid) [inline contributing.md]: We probably don't need a `<br>` here, I think leaving the line blank would give the same effect

File: contributing.md [hunk 4]
```diff
@@ -1,5 +1,7 @@
 <br>
```
Comment (galuhsahid) [inline contributing.md]: I see that this is from the previous PR, maybe we can remove the `br` here too?

File: contributing.md [hunk 5]
```diff
@@ -70,3 +72,26 @@
...
+* If yes and open refer to existing issue.
+* If yes and closed reopen issue with descriptive comment.
+* If no, create the issue by;
+   * Following our issue [template.](https://github.com/pandas-dev/pandas/blob/master/.github/ISSUE_TEMPLATE.md)
+### Existing issue
+* Read comments.
+* Find out if anyone is working on it, if no, offer to do it. If yes, see if you can be of help.
+<br>
+
+## Reporting bugs
```
Comment (sara-02) [inline contributing.md]: Reporting bugs should be a subcategory of the issues, please move it under the Issues. And make its header `###`

File: contributing.md [hunk 6]
```diff
@@ -70,3 +72,26 @@
 ### Reference
 * [GitHub forking](https://gist.github.com/Chaser324/ce0505fbed06b947d962)  
 * [Git cherry-pick](https://git-scm.com/docs/git-cherry-pick)
+<br>
+
+## PR review rules
+ * Input from two reviewers is needed to merge a pull request. One reviewer approves the merge and the other reviewer merges the pull request.
+<br>
+
+## Issues
```
Comment (sara-02) [inline contributing.md]: Maybe label the heading as  `Issue Guidelines`?

File: contributing.md [hunk 7]
```diff
@@ -70,3 +72,26 @@
 ### Reference
 * [GitHub forking](https://gist.github.com/Chaser324/ce0505fbed06b947d962)  
 * [Git cherry-pick](https://git-scm.com/docs/git-cherry-pick)
+<br>
+
+## PR review rules
+ * Input from two reviewers is needed to merge a pull request. One reviewer approves the merge and the other reviewer merges the pull request.
+<br>
```
Comment (sara-02) [inline contributing.md]: remove the `<br>` from here as well.

**Review by sara-02 — CHANGES_REQUESTED** [review]

**Comment by sara-02** [comment]
@Yomdroid Both I and @galuhsahid have requested a few changes, please look into them.

**Comment by yomaokobiah** [comment]
@sara-02 @galuhsahid changes made in  14e8e07

**Review by dujm — APPROVED** [review]

**Review by sara-02 — APPROVED** [review]

**Comment by sara-02** [comment]
@Yomdroid Thanks.

**Pull request merged**